scripts/fetch_lints.py

```diff
@@ -54,31 +53,13 @@ def normalize(name):
 
 
 def get_clippy_lints():
-    if not os.path.isdir("rust-clippy"):
-        subprocess.run(["git", "clone", "https://github.com/rust-lang/rust-clippy"],
-                       stdout=subprocess.DEVNULL,
-                       stderr=subprocess.DEVNULL,
-                       stdin=subprocess.DEVNULL,
-                       cwd=DIR,
-                       check=True)
-    else:
-        subprocess.run(["git", "pull"],
-                       stdout=subprocess.DEVNULL,
-                       stderr=subprocess.DEVNULL,
-                       stdin=subprocess.DEVNULL,
-                       cwd=os.path.join(DIR, "rust-clippy"),
-                       check=True)
-
-    sys.path.insert(0, os.path.join(DIR, "rust-clippy", "util"))
-    from lintlib import parse_all
-
-    clippy_lints, _ = parse_all(os.path.join(DIR, "rust-clippy", "clippy_lints", "src"))
+    clippy_lints = requests.get("http://rust-lang.github.io/rust-clippy/master/lints.json").json()
```

Comment:
Can we use `urllib` instead of external `requests`?
Otherwise, you have to install `requests` into your system python to run `updateLints` task that unnecessarily complicates updating lints at the first time

Reply:
I thought that `requests` is already a "required dependency", since it's in the [`Pipfile`](https://github.com/intellij-rust/intellij-rust/blob/master/scripts/Pipfile) and it's already used by [`release-actions.py`](https://github.com/intellij-rust/intellij-rust/blob/master/scripts/release-actions.py). I'll change it to urllib.

Reply:
The main difference that `release-actions.py` is supposed to be run directly
In case of `updateLints` task, python script is run by gradle that knows nothing about pipenv currently. It leads using system pythong interpreter

Reply:
Good point! Didn't think of that :) It's changed now.